core/themes/seven/css/style.css

```diff
@@ -890,6 +890,7 @@ td.checkbox .form-item input {
 }
 ul.tips,
 div.description,
+div.help,
```

Comment:
I come across this very often and get annoyed by the fact that individual help form elements are not styled the same as the help (description) text for all other elements. Without this change, things look like so:

<img width="662" alt="Screen Shot 2020-07-25 at 8 42 51 pm" src="https://user-images.githubusercontent.com/2423362/88462943-e1bfc580-ceb7-11ea-8b65-8360a7f53798.png">

Reply:
These should both be descriptions. In fact, they should be descriptions on the checkboxes, not separate elements. But let's not do this in this issue :) Can you create another?

Reply:
I was thinking the opposite - that help elements should be big and blue with a ? on the left (so they match info messages?)

The light gray text and smaller font for description elements is so they are less obvious and easier to skip over with the eye. At first glance, they should indicate to the viewer they are "optional" and don't need to be read. 

Help areas, on the other hand, should be big and bold and let the viewer know that they are there to assist with the tasks that follow. 

(I think this is why there is no style on help elements -- lets make a separate issue to decide what to do with them)